integration/spark/app/src/main/java/io/openlineage/spark/agent/ArgumentParser.java

```diff
@@ -37,26 +39,40 @@
 public class ArgumentParser {
 
   public static final String SPARK_CONF_NAMESPACE = "spark.openlineage.namespace";
+
   public static final String SPARK_CONF_PARENT_JOB_NAMESPACE =
       "spark.openlineage.parentJobNamespace";
+  
   public static final String SPARK_CONF_PARENT_JOB_NAME = "spark.openlineage.parentJobName";
+
   public static final String SPARK_CONF_PARENT_RUN_ID = "spark.openlineage.parentRunId";
+
   public static final String SPARK_CONF_APP_NAME = "spark.openlineage.appName";
+
   public static final String ARRAY_PREFIX_CHAR = "[";
   public static final String ARRAY_SUFFIX_CHAR = "]";
   public static final String DISABLED_FACETS_SEPARATOR = ";";
+
   public static final String SPARK_CONF_TRANSPORT_TYPE = "spark.openlineage.transport.type";
+
   public static final String SPARK_CONF_HTTP_URL = "spark.openlineage.transport.url";
+
   public static final String SPARK_CONF_JOB_NAME_APPEND_DATASET_NAME =
       "spark.openlineage.jobName.appendDatasetName";
+
   public static final String SPARK_CONF_JOB_NAME_REPLACE_DOT_WITH_UNDERSCORE =
       "spark.openlineage.jobName.replaceDotWithUnderscore";
+  
   private static final String SPARK_CONF_FACETS_DISABLED = "spark.openlineage.facets.disabled";
 
   private static final String SPARK_CONF_DEBUG_FACET = "spark.openlineage.debugFacet";
 
   private static final String SPARK_TEST_EXTENSION_PROVIDER =
       "spark.openlineage.testExtensionProvider";
+  
+  private static final String VAR_ENV_OPENLINEAGE_APP_NAME = "OPENLINEAGE__APP_NAME";
```

Comment:
For regular `OPENLINEAGE_` env vars I'd keep single `_` for consistency with rest of env vars, unless we have some good reason to not do that.